pkgs/development/libraries/nanopb/test-message-with-options/default.nix

```diff
@@ -1,9 +1,15 @@
-{ stdenv, protobuf, nanopb }:
+{ stdenv, protobuf, nanopb, lib }:
```

Comment:
here and elsewhere: lib usually goes at the start of these argument lists, but it doesn't matter.

Reply:
Fixed!